app/lib/actions/katello/content_view_version/import_library.rb

```diff
@@ -0,0 +1,71 @@
+# rubocop:disable Lint/SuppressedException
+module Actions
+  module Katello
+    module ContentViewVersion
+      class ImportLibrary < Actions::EntryAction
+        def plan(organization, path:, metadata:)
+          action_subject(organization)
+          unless SmartProxy.pulp_primary.pulp3_repository_type_support?(::Katello::Repository::YUM_TYPE)
+            fail ::Katello::HttpErrors::BadRequest, _("This API endpoint is only valid for Pulp 3 repositories.")
+          end
+          ::Katello::Pulp3::ContentViewVersion::Import.check!(content_view: organization.default_content_view, metadata: metadata, path: path)
+
+          version = organization.default_content_view_version
+          history = ::Katello::ContentViewHistory.create!(:content_view_version => version,
```

Comment:
nitpick, up to you if want to change because this is my personal style choice. this type of indentation is very jarring to me. I prefer:

```
history = ::Katello::ContentViewHistory.create!(
  :content_view_version => version,
  :user => ::User.current.login,
  ...
)
```

ie indenting based on the first character of the previous line, not the method arguments